src/test/com/arrary/Arraylist.java

```diff
@@ -0,0 +1,32 @@
+package test.com.arrary;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Створити ArrayList стрінгів numbers з назвами цифр від нуля до 10,
+ * потім використовуючи синтаксис for(String number : numbers) пройтись по ньому і вивести його в консоль.
+ * Друге завдання замінити 3 і 5 номера на цифри (просто як стрінги будуть але цифри замість слів). Перевірити результат виведенням в консоль
+ * Третє завдання видалити 2 і 9 цифри і вивести в консоль
+ */
+public class Arraylist {
+    public static void main(String[] args) {
+        List<String> numbers = new ArrayList<>(List.of("one", "two", "tree", "four", "five", "six", "seven", "eight", "nine", "ten"));
```

Comment:
I think that new ArrayList<> or List.of is redundant they do the same thing, isn't it?